dist_tables/dml.rst

```diff
@@ -8,23 +8,25 @@ The following code snippets use the Github events example, see :ref:`ddl`.
 Inserting Data
 --------------
 
-Single row inserts
-$$$$$$$$$$$$$$$$$$
+Inserting Row Literals
```

Comment:
@begriffs -- Instead of editing the example in this section, what do you think about providing another example?

I'd consider multi-row inserts an advanced feature. Most users would start by running single row insert statements. For example, I ran my first multi-row insert statement four years into my Postgres journey.